apps/web/pages/api/book/event.ts

```diff
@@ -555,6 +562,16 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       }
 
       await sendRescheduledEmails({ ...evt, additionInformation: metadata });
+
+      // Reschedule reminders
+      for (const reminder of eventType.attendeeReminders) {
+        if (reminder.method === "SMS" && evt.reminderPhone) {
+          await scheduleSMSAttendeeReminder(evt, evt.reminderPhone, reminder);
+        }
+        if (reminder.method === "EMAIL" && evt.reminderPhone) {
+          await scheduleEmailReminder(evt, evt.attendees[0].email, reminder);
+        }
+      }
```

Comment:
The event manager deals with deleting the previous scheduled reminders. We can then create new ones